### ISSUE TITLE
Support for ssl.com

### ISSUE THREAD

**Issue by einar-hjortdal** [candango/automatoes#120]
Does automatoes support issuing certificates with ssl.com?
I believe it works like zerossl, as it requires an account to be registered on the website and then bind the on-site account to the acme-created account.
I think this is called external account binding

**Comment by piraz** [comment]
Do they support ACMEv2?

**Comment by einar-hjortdal** [comment]
> Do they support ACMEv2?

Yes, they only support v2. They have 2 different api endpoints for ecdsa and rsa, more info here https://www.ssl.com/guide/ssl-tls-certificate-issuance-and-revocation-with-acme/

**Comment by piraz** [comment]
Yes it does: https://www.ssl.com/how-to/acme-ssl-tls-automation-with-apache-and-nginx/

It should work using `--server` paramter.

I think the register part will be tricky. You can use the `manuale migrate` command added by https://github.com/candango/automatoes/issues/116 to get the certbot key converted to the automatoes format.

I need to add some docs on that....

**Comment by einar-hjortdal** [comment]
> I think the register part will be tricky. You can use the `manuale migrate` command added by #116 to get the certbot key converted to the automatoes format.

Not migrating though, planning to use automatoes from scratch. With certbot I would just pass `--eab-hmac-key HMAC-KEY --eab-kid ACCOUNT-KEY`, don't know if it'll work with automatoes.

**Comment by piraz** [comment]
Wanna get me to give a try? I can see if this this should be turned into a feature.

What about `manuale register --server <rsa-server-option> --key <HMAC-KEY-FILE>`?

I think eab-kid is generated during the process, but if that does no work, I can get on that.

**Comment by einar-hjortdal** [comment]
> Wanna get me to give a try? I can see if this this should be turned into a feature.

I want to give it a try but I can't. My ISP doesn't let me and I do not currently have a server to try with. I want to know if it works before paying for a server

**Comment by piraz** [comment]
Sorry, that wasn't my question. Let me rephrase that.

I'll give a try, if does not work I'll make it work, and you can use it without problems.

Also it is good they have ec, maybe I can get that supported also.

**Comment by szepeviktor** [comment]
Here is how it works with Google Trust Services.
https://support.google.com/domains/answer/7630973?hl=en#zippy=%2Cget-an-ssltls-certificate-from-a-certificate-authority
